openapi/index.yaml

```diff
@@ -416,85 +416,91 @@ parameters:
     x-example: DOWNLOAD
 definitions:
   ExtendedProfile:
-    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v20.4.0/openapi/definitions.yaml#/ExtendedProfile"
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v20.6.1/openapi/definitions.yaml#/ExtendedProfile"
```

Comment:
Shouldn't all these be v20.6.5?

Reply:
@balanza @BurnedMarshal

Reply:
@michaeldisaro @balanza  in my PR #170 I've switched to openapi generation from index.openapi.template.
In this way we removed this reference by importing deinitions from fn-commons node_module (see here: https://github.com/pagopa/io-functions-app/blob/ccd26f65dfaa2a6e7f73bbcdc7b615388549697a/openapi/index.yaml.template#L458)

Reply:
Seems good. So we don't have to keep it updated anymore.